file1.txt

```diff
@@ -1,10 +1,10 @@
 1
-2
+2 = 1 + 1
 3
-4
+4  # FIXME: is this 2+2 or 2*2?
 5
 6
-7
+7 = 4 + 3
 8
-9
+9 = 8 + 7
```

Comment:
Please check your math.  

Also do another PR adding some lines like 2.1 2.2

Reply:
Please see 5e91562535 and ad868dd5eb300537